front/types/chat.ts

```diff
@@ -14,10 +14,9 @@ export type ChatRetrievedDocumentType = {
 
 export type ChatMessageType = {
   role: "user" | "retrieval" | "assistant" | "error";
-  runRetrieval?: boolean;
-  runAssistant?: boolean;
   message?: string; // for `user`, `assistant` and `error` messages
   retrievals?: ChatRetrievedDocumentType[]; // for `retrieval` messages
+  query?: string; // for `retrieval` messages (not persisted)
```

Comment:
Should we store at list client side for now the time picker value otherwise users won't have visibility on what was picked?

Reply:
As per IRL the time picker is not auto yet.

Reply:
as per irl query is only the query string for now; will turn it to a proper json for the auto time picking PR